### DISCUSSION TITLE
Option to choose begin/end time of the day

### DISCUSSION THREAD

**Discussion by ascsanto** [davish/obsidian-full-calendar#387] [Feature Requests]
My problem is that since i usually go to sleep at 2am, if i want to track also the 0am-2am part of the day, i don't have their events at first glance, i have to scroll up in the weekly view, and change day in the day view, which is very inconvenient and uncomfortable since i have to do this every day.
I don't know how much work is this since i didn't see the code, but it would be useful to add an option to choose the time when each day begins/ends.

**Comment by davish** [comment]
I'm not sure if it's possible to push the end of the day back past midnight. But if it is, it would be configured with `slotMinTime` and `slotMaxTime` settings ([documentation here](https://fullcalendar.io/docs/slotMaxTime)). Will require a bit of investigation.

**Comment by 0-ft** [comment]
I was able to achieve this effect by modifying `calendar.ts`, as you suspected @davish.
![image](https://user-images.githubusercontent.com/98317611/169624296-4c792d0b-f85e-460b-aff6-33a7028296b8.png)
By setting the `slotMaxTime` to some length > 24 hours e.g. 28 hours, the extra ours are shown at the end of the day. Adding events seems to work fine and they are displayed in both the after-hours section they were added and also the morning of the following day. By setting the day length to 24hr but offset by some amount of time we could probably achieve what OP wants.

This could be worked up into a settings option relatively easily.
